### DISCUSSION TITLE
Postgresql createdby modifiedby createdat modifiedat

### DISCUSSION THREAD

**Discussion by nev-21** [Budibase/budibase#6712] [Help] [answered]
Is there a possibility to add this fields when I create table in posgresql? or I have to do it manually?.

**Comment by melohagan** [comment, marked as answer]
Hey @nev-21 

For external datasources it has to be done manually. The auto-columns are for the internal Budibase DB only.

**Reply by nev-21** [reply]
Ok @melohagan. And thanks for budibase, is great.

**Reply by melohagan** [reply]
You're welcome - thanks for the kind words :)
